src/components/patientForm/patientForm.tsx

```diff
@@ -150,12 +156,14 @@ export const PatientForm: React.FC = () => {
         <Form.Item
           name="Confirm"
           valuePropName="checked"
-          wrapperCol={{ offset: 5, span: 16 }}
+          // wrapperCol={{ offset: 6, span: 10 }}
           rules={[{ required: true, message: "Confirmation Required" }]}
         >
-          <Checkbox>All the information I entered is true</Checkbox>
+          <Checkbox>Confirm all information</Checkbox>
```

Comment:
Are we checking the purpose of this checkBox? Maybe validate the submit button so it gets clicked only after user has confirmed this checkbox.

Reply:
Thank you, I will look into it!